estate/models/estate_property_type.py

```diff
@@ -7,4 +7,5 @@ class EstatePropertyType(models.Model):
     _description = "estate property type"
     _order = "id"
 
-    name = fields.Char("Name", required=True)
\ No newline at end of file
+    name = fields.Char("Name", required=True)
+    
```

Comment:
The warning is still there... I guess your last line is not empty... maybe it contains a few spaces. (This happens in many files)